src/main/java/duke/util/Parser.java

```diff
@@ -0,0 +1,38 @@
+package duke.util;
+
+import duke.command.*;
```

Comment:
Try to configure your IDE to not optimise imports. Imports (at least in this module) should always be explicitly stated. 
![image](https://user-images.githubusercontent.com/77244987/131905115-1859339f-a29d-4bde-8f73-863a6f7c4f80.png)